src/calculateRentalCost.js

```diff
@@ -28,7 +28,10 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discount = days >= 3 ? days >= 7 ? 50 : 20 : 0;
```

Comment:
The checklist suggests that you don't use literals in expressions (point 2), instead of writing `3` create a variable above it: ```const SMALL_DISCOUNT_REQUIREMENT = 3;``` and use that in your expression, so others understand what the number means.